apps/studio/schemas/objects/content.tsx

```diff
@@ -24,6 +24,7 @@ export const content: ArrayField = {
     { type: "imports" },
     { type: "imageWithCaption" },
     { type: "codeExample" },
+    { type: "staticCodeBlock" },
```

Comment:
Det _er_ litt klønete at vi nå har to forskjellige kodeblokk-komponenter. Men jeg tror ikke vi slipper unna – de har såpass forskjellige bruksområder at det ville vært vanskelig å tilpasse én til å gjøre begge jobbene. Vi får heller iterere på navngivningen.